flink-table/flink-table-common/src/main/java/org/apache/flink/table/types/logical/utils/LogicalTypeUtils.java

```diff
@@ -36,6 +43,54 @@ public static LogicalType removeTimeAttributes(LogicalType logicalType) {
 		return logicalType.accept(TIME_ATTRIBUTE_REMOVER);
 	}
 
+	/**
+	 * Get internal(sql engine execution data formats) conversion class for {@link LogicalType}.
```

Comment:
nit: `Returns the conversion class for the given {@link LogicalType} that is used by the table runtime.`